validator/rpc/server.go

```diff
@@ -170,50 +169,49 @@ func (s *Server) InitializeRoutes() error {
 		return errors.New("no router found on server")
 	}
 	// Adding Auth Interceptor for the routes below
-	s.router.Use(s.AuthTokenHandler)
+	s.AuthTokenHandler(s.router)
```

Comment:
I don't think this will work. i don't think this will actually apply the auth token handler to the router